Source/SwiftLintBuiltInRules/Rules/Idiomatic/VoidFunctionInTernaryConditionRule.swift

```diff
@@ -199,6 +307,37 @@ private extension CodeBlockItemSyntax {
 
         return parent.children(viewMode: .sourceAccurate).count == 1
     }
+
+    // For codeBlockItem, recursively traverse it to determine if it is within its own FunctionDeclSyntax.
+    func getFunctionDeclSyntax(codeBlockItem: CodeBlockItemSyntax) -> FunctionDeclSyntax? {
+      let targetSyntax = codeBlockItem.parent?.parent?.parent?.parent
+      if let targetSyntax = targetSyntax?.as(FunctionDeclSyntax.self) {
```

Comment:
This covers only functions, but there are also the other cases which are checked above in the `is...Return` properties.